### ISSUE TITLE
Test against third-party libraries

### ISSUE THREAD

**Issue by frenic** [frenic/csstype#13]
Even small changes may [have impact on third-party libraries](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/23116#issuecomment-370103160) that uses CSSType (like JSS, Styletron and Glitz) and tests against these would provide safety when publishing new releases. Not sure about the best approach to keep it quite general for both TypeScript/Flow and when API/definitions/tests changes on these libraries? It shouldn't fail unless a change in CSSType is the cause of the failure.

**Comment by pelotom** [comment]
Add React to that list now!

**Comment by frenic** [comment]
Yeah, we don't want that to happen again 😅 I'll do it manually until I figure out a way to automate it.

**Comment by frenic** [comment]
Hm, it seems like it doesn't work just by linking `csstype` and run `yarn test` locally with DefinitelyTyped. It seems like `dtslint` makes clean installations of NPM dependencies before linting. `yarn lint react` works though but I would like to cover other packages that depends on `React.CSSProperties` as well.

**Comment by estaub** [comment]
Please consider adding Material-UI to the list.  
I know it's indirect, via JSS, but still.

**Comment by frenic** [comment]
@estaub I know you had troubles when the JSS types was changed. But those were/are directly due to `@types/jss` and not `csstype` as I understand. Does `material-ui` have any types that relies on the properties and values from `csstype`? Could you point me into the right direction so I can get an understanding of which changes in `csstype` could possibly affect `material-ui`?
